cfgov/unprocessed/js/routes/on-demand/expandable-group.js

```diff
@@ -2,4 +2,45 @@
    Scripts for Expandable Group organism.
    ========================================================================== */
 
-require( 'cf-expandables/src/Expandable' ).init();
+import {
+  addEventListenerToSelector,
+  analyticsLog,
+  removeEventListenerFromSelector
+} from '../../../apps/analytics-gtm/js/util/analytics-util';
+
+import Expandable from 'cf-expandables/src/Expandable';
+Expandable.init();
+
+
+/**
+ * Tells Mouseflow to stop capturing, if it already is, and start a new capture.
+ */
+function stopStartMouseflow() {
+  console.log( 'stopStartMouseflow' );
```

Comment:
Why not use `analyticsLog( 'stopStartMouseflow' );` here and elsewhere where there are console.log statements?

Reply:
Ohh I guess this is still a draft.

Reply:
Yeah, I just left those in for ease of testing locally that the code paths are doing what we want, and then we'll test with Mouseflow itself next week.